recipe/build.sh

```diff
@@ -53,10 +53,19 @@ cp make.inc-examples/make.inc.openblas ./make.inc
 # standard supported by both tool sets.
 export CXXFLAGS="$(echo $CXXFLAGS | sed -e 's/ -std=[^ ]*//') -std=c++14"
 
+GPU_TARGET="sm_37 sm_60 sm_70 sm_75"
```

Comment:
I eventually want to pull this out to a central location, but for now lets standardize on:

CUDA10.2:
x86:
    CUDA_OPTION_1='sm_37,sm_52,sm_60,sm_61,sm_70,compute_75'

ppc64le:
    CUDA_OPTION_1='sm_37,sm_60,sm_70,compute_75'

CUDA11:
x86:
    CUDA_OPTION_1='sm_37,sm_52,sm_60,sm_61,sm_70,sm_75, compute_80'

ppc64le:
    CUDA_OPTION_1='sm_37,sm_60,sm_70,sm_75, compute_80'

Reply:
magma is internally expanding supplied `sm_*` flags to:

`-- CUDA_NVCC_FLAGS: -DHAVE_CUBLAS;-Xfatbin;-compress-all;-Xcompiler;-fPIC;-std=c++11;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_80,code=compute_80;-DADD_`